FirebaseRemoteConfig/Sources/FIRRemoteConfig.m

```diff
@@ -613,4 +623,18 @@ - (FIRConfigUpdateListenerRegistration *)addOnConfigUpdateListener:
   return [self->_configRealtime addConfigUpdateListener:listener];
 }
 
+#pragma mark - Rollout
+
+- (void)addRemoteConfigInteropSubscriber:(id<FIRRolloutsStateSubscriber>)subscriber {
+  //    [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(subscriber) name:
+  //    RolloutsStateDidChangeNotificationName object:nil];
+  [[NSNotificationCenter defaultCenter]
+      addObserverForName:RolloutsStateDidChangeNotificationName
+                  object:nil
```

Comment:
I tend to use `self` rather than nil. Imaging if you have multiple remoteConfig instances, if you don't specify which object is the sender of your notification, then every RC instance rollouts update will trigger all subscribers(no matter in which instance).